wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectService.java

```diff
@@ -351,8 +361,16 @@ public void importProject(@ApiParam(value = "Path in the project", required = tr
                                                                      ServerException,
                                                                      NotFoundException,
                                                                      BadRequestException {
-        projectManager.importProject(path, sourceStorage, force,
-                                     () -> new ProjectImportOutputWSLineConsumer(path, workspace, 300));
+
+        final int delayBetweenMessages = 300;
+
+        final ProjectImportOutputWSLineConsumer wsLineConsumer =
```

Comment:
Yes, websocket line consumer still needs for dashboard uses. Left for backward compatibility.